app/admin/concepts/statistic/operation.rb

```diff
@@ -0,0 +1,45 @@
+class StatisticOperation
+  class EventCount
+    class Show
+      def call(params = {})
+        Event.count
+      end
+    end
+  end
+
+  class TotalParticipants
```

Comment:
I'll update this tomorrow. If I have time. Tomorrow is the Chatbot Summit that I stayed here so long for.